packages/store-indexer/bin/postgres-decoded-indexer.ts

```diff
@@ -15,7 +15,8 @@ const env = parseEnv(
   z.intersection(
     indexerEnvSchema,
     z.object({
-      DATABASE_URL: z.string(),
```

Comment:
can we keep this for backwards compat?

Reply:
Yes. I was worried about readability long term